repo/packages/H/hdfs/0/config.json

```diff
@@ -11,7 +11,8 @@
           "type": "string"
         },
         "default": [
-          "https://s3.amazonaws.com/downloads.mesosphere.io/hdfs/artifacts/dcos/0.1.1/hdfs-mesos-0.1.1.tgz"
+          "https://s3.amazonaws.com/downloads.mesosphere.io/hdfs/artifacts/dcos/0.1.1/hdfs-mesos-0.1.1.tgz",
+          "https://downloads.mesosphere.io/java/jre-7u76-linux-x64.tar.gz"
         ]
```

Comment:
Are you referring to "https://downloads.mesosphere.io/java/jre-7u76-linux-x64.tar.gz" when you are speaking about not hard coding here?  This change is in the config file.